diff -Nru librep-0.13/configure.in librep-0.13.new/configure.in
--- librep-0.13/configure.in	Wed Sep 20 21:11:43 2000
+++ librep-0.13.new/configure.in	Tue Sep 26 07:22:07 2000
@@ -53,8 +53,8 @@
 esac
 
 repdir='${datadir}/rep'
-replispdir='${repdir}/${version}/lisp'
-repexecdir='${libexecdir}/rep/${version}/${host_type}'
+replispdir='${repdir}/lisp'
+repexecdir='${libexecdir}/rep/${host_type}'
 repcommonexecdir='${libexecdir}/rep/${host_type}'
 repdocfile='${repexecdir}/DOC'
 
