--- librep-0.17/rep.m4.orig	1999-11-19 14:50:26.000000000 +0000
+++ librep-0.17/rep.m4	2004-03-15 18:37:05.000000000 +0000
@@ -4,7 +4,7 @@
 dnl AM_PATH_REP([MINIMUM_VERSION])
 dnl Test for librep, define REP_VERSION, REP_CFLAGS, REP_LIBS and REP_EXECDIR
 dnl
-AC_DEFUN(AM_PATH_REP,
+AC_DEFUN([AM_PATH_REP],
 [dnl
   AC_ARG_WITH(rep_prefix,[  --with-rep-prefix=PFX   Prefix where rep is installed (optional)],
 	      [rep_prefix="$withval"], [rep_prefix=""])
